assets/theme-css/figure.css

```diff
@@ -0,0 +1,43 @@
+img {
+  max-width: 100%;
+}
+
+img.align-left,
+figure.align-left,
```

Comment:
Do we parse this file with SCSS yet? If so, something like this might be slightly easier to read:

```css

#header,
#header-g {
    .l-header-top {
       height: 55px;
       line-height: 50px;
       border-top: 5px solid #474747;
       background-color: #f0f1f3;
       font-size: 16px;
    }
}
```

Reply:
We can refactor the old css to scss in a future PR.